flow/project.py

```diff
@@ -4678,7 +4714,7 @@ class MyProject(FlowProject):
         execution_group.add_argument(
             "-t",
             "--timeout",
-            type=int,
+            type=float,
```

Comment:
Is there a reason to support fractional timeouts? I'm fine with it, just wondering what you intended here.